docs/src/modules/components/prism.js

```diff
@@ -15,7 +15,7 @@ let darkTheme;
 
 if (process.browser) {
   lightTheme = require('prismjs/themes/prism.css');
-  darkTheme = require('prismjs/themes/prism-okaidia.css');
+  darkTheme = require('prismjs/themes/prism-tomorrow.css');
```

Comment:
Nice!

Reply:
Switched it out again for a better (in terms of contrast ratio) one. Light theme doesn't quite hit the 4.5 but lowest value is 4.4 for ghcolors.css which is way better than 3.0

Reply:
What do you think of always using the dark theme for the demos?